osquery/carver/carver.cpp

```diff
@@ -167,14 +168,40 @@ void Carver::start() {
     carvedFiles.insert(fs::path(p));
   }
 
-  auto s = compress(carvedFiles);
+  auto s = archive(carvedFiles, archivePath_);
   if (!s.ok()) {
     VLOG(1) << "Failed to create carve archive: " << s.getMessage();
     updateCarveValue(carveGuid_, "status", "ARCHIVE FAILED");
     return;
   }
 
-  s = postCarve(archivePath_);
+  fs::path upload;
```

Comment:
nit: Would you mind making this `uploadPath` so as to match the variables above, and make understanding what this is a little more clear? I'm looking specifically at the `postCarve(upload)`